### ISSUE TITLE
Disable font size value added in the accessibility configuration 

### ISSUE THREAD

**Issue by BrianCortes** [lawnstarter/react-native-picker-select#461]
**Is your feature request related to a problem? Please describe.**<br>
I was asking if there is a possibility to add a new feature that allow me disabled the font size value added in the accessibility configuration on my device, because currently the `react-native-picker-select` doesn’t have this functionality. this feature already exist in react native, the Text component has this prop `allowFontScaling` , I don't know if exist any possibility to it.

##### Example android configuration to reproduce the error:
Go to accessibility section in the settings and then select the Font size opcion.
![image](https://user-images.githubusercontent.com/18728297/136847498-8c94e94d-4e7c-4e77-a407-92a606e9d1be.png)

##### Example visual issue:
In this case location are using `react-native-picker-select` 
![image](https://user-images.githubusercontent.com/18728297/136847423-66d557fb-dc54-4212-a09e-7c62970c576f.png)

**Describe the solution you'd like**<br>
Add a prop allowFontScaling in the component to disabled the font size value added in the accessibility configuration on my device

**Describe alternatives you've considered**<br>

**Additional details**<br>
Add any other details or screenshots about the feature request here.
